tests/test_batch_change.py

```diff
@@ -76,27 +83,35 @@ def test_get_batch_change(mocked_responses, vinyldns_client):
 
     drc = DeleteRecordSetChange(forward_zone.id, forward_zone.name, 'baz', 'baz.bar.com', RecordType.A, 'Complete',
                                 'id2', 'system-message', 'rchangeid2', 'rsid2')
-    bc = BatchChange('user-id', 'user-name', 'batch change test', datetime.utcnow(), [arc, drc],
-                     'bcid', 'owner-group-id')
+
+    bc = BatchChange('user-id', 'user-name', datetime.utcnow(), [arc, drc],
+                     'bcid', 'PendingProcessing', 'AutoApproved',
```

Comment:
This looks like bad data since the batch change `Status` should be `Complete` if all `SingleChange`s are complete.